screens/ExpensesScreen/BarChart.jsx

```diff
@@ -19,7 +20,9 @@ const BarChart = ({data}) => {
     </View>
```

Comment:
I provided example for this part 

   renderItem={({item, index}) => {
            return <ExpenseItem item={item} index={index} />;
          }}